### PR TITLE
ci(docker): use public.ecr.aws registry

### PR DESCRIPTION
**Pull request by id**
Fixes docker hub rate limiting.

### CODE REVIEW

**Review by thalesmg — DISMISSED** [review]

**Review by zmstone — DISMISSED** [review]

File: .ci/docker-compose-file/docker-compose-cassandra.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: '3.9'
 
 x-cassandra: &cassandra
   restart: always
-  image: cassandra:${CASSANDRA_TAG:-3.11.6}
+  image: public.ecr.aws/docker/library/cassandra:${CASSANDRA_TAG:-3.11}
```
Comment (keynslug) [inline .ci/docker-compose-file/docker-compose-cassandra.yaml:5]:
```
Error response from daemon: manifest for public.ecr.aws/docker/library/cassandra:3.11.6 not found: manifest unknown: Requested image not found
```

File: .ci/docker-compose-file/docker-compose-minio-tcp.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ version: '3.7'
 services:
   minio:
     hostname: minio
-    image: quay.io/minio/minio:${MINIO_TAG}
+    image: public.ecr.aws/bitnami/minio:${MINIO_TAG}
```
Comment (keynslug) [inline .ci/docker-compose-file/docker-compose-minio-tcp.yaml:6]:
```
Error response from daemon: manifest for public.ecr.aws/bitnami/minio:RELEASE.2023-03-20T20-16-18Z not found: manifest unknown: Requested image not found
```

**Review by keynslug — COMMENTED** [review]
Only those 2 are missing in ECR it seems.

**Comment by zmstone** [comment]
merged to release-53 since this is the only PR to master since code-freeze

**Pull request merged**